app/renderer/js/notification/helpers.js

```diff
@@ -83,6 +83,36 @@ function parseReply(reply) {
 		reply = reply.replace(regex, streamMention);
 	});
 
+	window.$.ajaxSetup({async: false});
```

Comment:
Right here blocking the thread using synchronous request is not the right approach here